### PR TITLE
Issues3,12,23,24

### PR DESCRIPTION
**Pull request by Lightning0**
If you guys could review this, I would appreciate it. I know I am not completely done, but I wanted to have something to show.

### CODE REVIEW

File: GameMaker.py [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
-	def create_landmark(self):
+	def create_landmark(self, name, place) -> bool:
 		pass
 
-	def edit_landmark_clue(self):
+	def edit_landmark_clue(self, name, old_clue, new_clue) -> bool:
 		pass
 
-	def edit_landmark_question(self):
+	def edit_landmark_question(self, name, old_question, new_question) -> bool:
```
Comment (teusday) [inline GameMaker.py]: Creating a question should require an answer

File: Tests/GameMakerTest.py [hunk 2]
```diff
@@ -3,6 +3,40 @@
...
 class TestGameMaker(unittest.TestCase):
-
 	def setUp(self):
 		self.GM = GameMaker()
+
+	# Not sure if this is a proper test: TODO: Verify Test_start
+
+	def test_start(self):
+		self.this_game = self.GM.create_game()
+		self.assertTrue(self.this_game.is_on(), "The game has not been created.")
```
Comment (teusday) [inline Tests/GameMakerTest.py]: Game isn't on until gamemaker starts game

File: Tests/GameMakerTest.py [hunk 3]
```diff
@@ -3,6 +3,40 @@
...
+
+	def test_create_landmark(self):
+		self.assertTrue(self.GM.create_landmark("name", "place"), "The landmark has not been created")
+
+	def test_create_landmark_duplicate(self):
+		self.GM.create_landmark("name", "place")
+		self.assertFalse(self.GM.create_landmark("name", "place"), "The landmark has been created twice.")
```
Comment (teusday) [inline Tests/GameMakerTest.py]: Not only should this be false, but you should also check that it didn't add a landmark and still return false. Might be an integration test more than a unit test?

File: GameMaker.py (hunk 1 above)
Comment (Lightning0) [reply GameMaker.py]: I changed the signature to ->
	def create_landmark(self, name, clue, question, answer) -> bool:

File: Tests/GameMakerTest.py (hunk 2 above)
Comment (Lightning0) [reply Tests/GameMakerTest.py]: The game maker doesn't have a method to do that, so I assumed that it happens when he creates the game. If we add the method to the design—which I am cool with—I will update the test. 
I added a # self.this_game.start() to the test that is commented out. 

P.S. I did think of this, but I see now that I might have misunderstood the GameGameMaker interface. ould we resolve this in the lab?

File: GameMaker.py (hunk 1 above)
Comment (teusday) [reply GameMaker.py]: I mean there should be an "answer" argument to replace the question

**Comment by Lightning0** [comment]
If someone could review and merge this I would appreciate it.

**Pull request merged**